caf-correlation-jservlet/src/main/java/com/github/cafapi/correlation/jservlet/CorrelationIdFilter.java

```diff
@@ -45,11 +47,11 @@ public void doFilter(
     {
         final HttpServletRequest req = (HttpServletRequest) request;
         final HttpServletResponse resp = (HttpServletResponse) response;
-        final String correlationId = Optional.ofNullable(req.getHeader(CorrelationIdConfigurationConstants.HEADER_NAME))
+        final String correlationId = Optional.ofNullable(req.getHeader(HEADER_NAME))
             .filter(s -> !s.isEmpty())
             .orElseGet(() -> UUID.randomUUID().toString());
-        MDC.put(CorrelationIdConfigurationConstants.MDC_KEY, correlationId);
-        resp.addHeader(CorrelationIdConfigurationConstants.HEADER_NAME, correlationId);
+        MDC.put(MDC_KEY, correlationId);
+        resp.addHeader(HEADER_NAME, correlationId);
```

Comment:
The changes to this module are unrelated to the Spring changes that are the point of this PR.  They should really be in their own PR.  PRs should be focused on what they are doing, not contain a mixed bag of changes.